unittest/src/fake/leds/mod.rs

```diff
@@ -7,10 +7,26 @@
 use core::cell::Cell;
 use libtock_platform::{CommandReturn, ErrorCode};
 
+// -----------------------------------------------------------------------------
+// Driver number and command IDs
+// -----------------------------------------------------------------------------
+
+const DRIVER_NUM: u32 = 2;
+
+// Command numbers
+const DRIVER_CHECK: u32 = 0;
+const LED_ON: u32 = 1;
+const LED_OFF: u32 = 2;
+const LED_TOGGLE: u32 = 3;
+
 pub struct Leds<const LEDS_COUNT: usize> {
     leds: [Cell<bool>; LEDS_COUNT],
 }
 
+// -----------------------------------------------------------------------------
+// Implementation details below
+// -----------------------------------------------------------------------------
```

Comment:
This `impl` block is not an implementation detail, it is the public API for `Leds`, so this comment is incorrect (or misplaced).